### PR TITLE
Docs: Improve the create pull request guide for contributing

### PR DESCRIPTION
**Pull request by axelavargas**
**What this PR does / why we need it**:

It was missing in the contribution guide the rule of not using snapshots for unit tests on the frontend side.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: contribute/create-pull-request.md
Comment (marcusolsson) [inline contribute/create-pull-request.md:46]:
```suggestion
Pull requests that add or modify unit tests that are written in Jest must:
```

File: contribute/create-pull-request.md
Comment (marcusolsson) [inline contribute/create-pull-request.md:48]: Why "Avoid adding" and not "Don't add"? Are we looking to completely remove snapshot tests, or are there still valid use cases?

File: contribute/create-pull-request.md
Comment (hugohaggmark) [inline contribute/create-pull-request.md:49]: Maybe this is better:
```suggestion
- If an existing unit test is written in enzyme, migrate it to RTL (React Testing Library), except if you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
```

Comment (Clarity-89) [inline contribute/create-pull-request.md:49]:
```suggestion
- If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), except if you’re “just” fixing an issue before a major release (i.e v8), then it’s ok to skip migrating the test to RTL
```

**Review by hugohaggmark — APPROVED** [review]
LGTM! But let's wait for the technical writer's approval.

File: contribute/create-pull-request.md
Comment (Clarity-89) [inline contribute/create-pull-request.md:46]: Should it maybe be clarified that this applies to front end tests specifically?

File: contribute/create-pull-request.md
Comment (marcusolsson) [inline contribute/create-pull-request.md:49]:
```suggestion
- If an existing unit test is written in enzyme, then migrate it to RTL (React Testing Library). If you’re “just” fixing an issue before a major release (i.e v8), then it’s ok to skip migrating the test to RTL.
```

**Review by Clarity-89 — APPROVED** [review]
Great idea to add this to the docs! 🎉

File: contribute/create-pull-request.md
Comment (hugohaggmark) [reply contribute/create-pull-request.md:48]: We're currently migrating from Enzyme to RTL one test at the time.

File: contribute/create-pull-request.md
Comment (axelavargas) [reply contribute/create-pull-request.md:46]: 🤔 I think because is on the ### Frontend-specific guidelines section does not need to be specific

Comment (Clarity-89) [reply contribute/create-pull-request.md:46]: Good point 👍

File: contribute/create-pull-request.md
Comment (marcusolsson) [reply contribute/create-pull-request.md:48]: I'm not sure I understand how this answers the question? If we don't allow new snapshot tests at all, this should say "Don't add" rather than "Avoid". The latter implies that it's ok to add snapshot tests in some circumstances.

File: contribute/create-pull-request.md
Comment (axelavargas) [inline contribute/create-pull-request.md:49]:
```suggestion
- If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), except if you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.
```

File: contribute/create-pull-request.md
Comment (axelavargas) [reply contribute/create-pull-request.md]: @marcusolsson I commit some suggestions, if you can take a second look at this point will be awesome 🙌🏾

File: contribute/create-pull-request.md
Comment (axelavargas) [reply contribute/create-pull-request.md:48]: yes, it can be confusing the use of 'avoid adding', I will change it to 'Don't add', because we don't want new snapshots :)

File: contribute/create-pull-request.md
Comment (axelavargas) [reply contribute/create-pull-request.md]: Thank you for the review @Clarity-89 🙌🏾  I made the changes :)

File: contribute/create-pull-request.md
Comment (oddlittlebird) [inline contribute/create-pull-request.md:46]:
```suggestion
Pull requests that add or modify unit tests that are written in Jest must adhere to these guidelines:
```

File: contribute/create-pull-request.md
Comment (oddlittlebird) [inline contribute/create-pull-request.md:48]:
```suggestion
- Don't add snapshots tests. We are incrementally removing existing snapshot tests, we don't want more.
```

File: contribute/create-pull-request.md
Comment (oddlittlebird) [inline contribute/create-pull-request.md:49]:
```suggestion
- If an existing unit test is written in Enzyme, migrate it to RTL (React Testing Library), unless you’re fixing a bug. Bug fixes usually shouldn't include any bigger refactoring, so it’s ok to skip migrating the test to RTL.

```

**Review by oddlittlebird — APPROVED** [review]
A couple minor edits, then good to go.

**Comment by axelavargas** [comment]
> A couple minor edits, then good to go.

Thank you @oddlittlebird 🙌🏾

**Pull request merged**
